.travis.yml

```diff
@@ -2,7 +2,7 @@ language: node_js
 node_js:
   - "7"
```

Comment:
node 7 is pretty old now...maybe better to leave out the version so it just always uses the latest stable?